### DISCUSSION TITLE
Cannot load collection into memory - Error ["failed to get replica info"]

### DISCUSSION THREAD

**Discussion by saratkirana** [milvus-io/milvus#30235] [Q&A and General discussion]
Hello, 

Milvus - 2.3.3 
No. of collections - 17  (13 in 1 database and 4 in another one)
Cluster 

I created a IVF_FLAT index on a 10 dim vector collection with 73k entities. I was able to load this before but after I created a big collection (487 M, 512 dim) and scaled query nodes to 35 (8 core, 64 GB). Loading stuck at 0%. After that even this smaller collection is also not getting loaded. I now scaled down the query nodes to 4 to test this small collection, even then it hangs at 0% loaded for a while and errors out. 

I can see this error in QeuryCoord 

`[2024/01/24 01:24:00.862 +00:00] [WARN] [querycoordv2/services.go:821] ["failed to get replica info"] [traceID=96f4baa1410c36a22d79eb60d1a260ea] [collectionID=446032668450742728] [replica=447208019517505549] [error="failed to get channels, collection not loaded: collection=446032668450742728: collection not found"] [errorVerbose="failed to get channels, collection not loaded: collection=446032668450742728: collection not found\n(1) attached stack trace\n  -- stack trace:\n  | github.com/milvus-io/milvus/pkg/util/merr.WrapErrCollectionNotFound\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/merr/utils.go:374\n  | [...repeated from below...]\nWraps: (2) failed to get channels, collection not loaded\nWraps: (3) attached stack trace\n  -- stack trace:\n  | github.com/milvus-io/milvus/pkg/util/merr.wrapWithField\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/merr/utils.go:760\n  | github.com/milvus-io/milvus/pkg/util/merr.WrapErrCollectionNotFound\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/merr/utils.go:372\n  | github.com/milvus-io/milvus/internal/querycoordv2.(*Server).fillReplicaInfo\n  | \t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/handlers.go:320\n  | github.com/milvus-io/milvus/internal/querycoordv2.(*Server).GetReplicas\n  | \t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/services.go:819\n  | github.com/milvus-io/milvus/internal/distributed/querycoord.(*Server).GetReplicas\n  | \t/go/src/github.com/milvus-io/milvus/internal/distributed/querycoord/service.go:385\n  | github.com/milvus-io/milvus/internal/proto/querypb._QueryCoord_GetReplicas_Handler.func1\n  | \t/go/src/github.com/milvus-io/milvus/internal/proto/querypb/query_coord.pb.go:5560\n  | github.com/milvus-io/milvus/pkg/util/interceptor.ServerIDValidationUnaryServerInterceptor.func1\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/interceptor/server_id_interceptor.go:54\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | github.com/milvus-io/milvus/pkg/util/interceptor.ClusterValidationUnaryServerInterceptor.func1\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/interceptor/cluster_interceptor.go:48\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | github.com/milvus-io/milvus/pkg/util/logutil.UnaryTraceLoggerInterceptor\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/logutil/grpc_interceptor.go:23\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc.UnaryServerInterceptor.func1\n  | \t/go/pkg/mod/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.38.0/interceptor.go:342\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:34\n  | github.com/milvus-io/milvus/internal/proto/querypb._QueryCoord_GetReplicas_Handler\n  | \t/go/src/github.com/milvus-io/milvus/internal/proto/querypb/query_coord.pb.go:5562\n  | google.golang.org/grpc.(*Server).processUnaryRPC\n  | \t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:1345\n  | google.golang.org/grpc.(*Server).handleStream\n  | \t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:1722\n  | google.golang.org/grpc.(*Server).serveStreams.func1.2\n  | \t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:966\n  | runtime.goexit\n  | \t/usr/local/go/src/runtime/asm_amd64.s:1598\nWraps: (4) collection=446032668450742728\nWraps: (5) collection not found\nError types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) merr.milvusError"]`

Any ideas what to do ? I checked datanode and datacord + Pulsar and Etcd logs. All of them look fine.

**Comment by saratkirana** [comment]
I can see the collection and everything in Attu. just loading does not work. If I create a new collection now and load, that works. Existing collections cannot be loading with this error. Also tried bringing the queryNode replicas to 1, still running into same issue.

**Comment by yhmo** [comment]
Use the client script to print all collections details:
```
for name in utility.list_collections():
    collection = Collection(name)
    print(collection.describe())
```

Is there a collection with collection_id=446032668450742728 ?

**Reply by saratkirana** [reply]
@yhmo 

Sorry for miscommunication. No. of collections - 17 (13 in 1 database and 4 in another one). Biggest collection has 487M entries and smallest is 73k entities. I was able to load the smallest collection fine before I created the big 487 M collection and scaled querynodes to 35. After that I am not able to load already created collections. To test I created a 3k entities (hello_milvus.py example and that works fine). Here is the collections list

`database - default 

{'collection_name': 'h_left_all_pinch', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668450742728, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_left_all_smoothed_pinch', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668547585435, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_right_all_temporal_frame', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 20}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668547798639, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_right_all_frame', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668498578494, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_right_all_pinch', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668498578509, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_right_all_smoothed_frame', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668522982430, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_right_all_smoothed_pinch', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668547585443, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'hello_milvus', 'auto_id': False, 'num_shards': 1, 'description': 'hello_milvus is the simplest demo to introduce the APIs', 'fields': [{'field_id': 100, 'name': 'pk', 'description': '', 'type': 21, 'params': {'max_length': 100}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'random', 'description': '', 'type': 11, 'params': {}, 'element_type': 0}, {'field_id': 102, 'name': 'embeddings', 'description': '', 'type': 101, 'params': {'dim': 8}, 'element_type': 0}], 'aliases': [], 'collection_id': 447208014526912886, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_left_all_temporal_frame', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 20}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668547798623, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_left_all_temporal_pinch', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 20}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668547798631, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_right_all_temporal_pinch', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 20}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668547798647, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_left_all_frame', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668450742721, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'h_left_all_smoothed_frame', 'auto_id': False, 'num_shards': 1, 'description': 'Loading H embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'embedding', 'description': '', 'type': 101, 'params': {'dim': 10}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668474546528, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}


database ve_ray

{'collection_name': 'testCollection', 'auto_id': False, 'num_shards': 1, 'description': 'trest', 'fields': [{'field_id': 100, 'name': 'asd', 'description': 'asd', 'type': 5, 'params': {}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'asdkjashd', 'description': 'akjshdas', 'type': 101, 'params': {'dim': 128}, 'element_type': 0}], 'aliases': [], 'collection_id': 447208014526482518, 'consistency_level': 2, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'hcb_ve_test1', 'auto_id': False, 'num_shards': 1, 'description': 'HCB TEST VE embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'offset', 'description': '', 'type': 5, 'params': {}, 'element_type': 0}, {'field_id': 102, 'name': 'original_url', 'description': '', 'type': 21, 'params': {'max_length': 65535}, 'element_type': 0}, {'field_id': 103, 'name': 'original_width', 'description': '', 'type': 3, 'params': {}, 'element_type': 0}, {'field_id': 104, 'name': 'original_height', 'description': '', 'type': 3, 'params': {}, 'element_type': 0}, {'field_id': 105, 'name': 'modi', 'description': '', 'type': 101, 'params': {'dim': 512}, 'element_type': 0}], 'aliases': [], 'collection_id': 446032668613378611, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'hcb_ve_test_ask_4', 'auto_id': False, 'num_shards': 1, 'description': 'HCB TEST VE embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'offset', 'description': '', 'type': 5, 'params': {}, 'element_type': 0}, {'field_id': 102, 'name': 'original_url', 'description': '', 'type': 21, 'params': {'max_length': 65535}, 'element_type': 0}, {'field_id': 103, 'name': 'original_width', 'description': '', 'type': 3, 'params': {}, 'element_type': 0}, {'field_id': 104, 'name': 'original_height', 'description': '', 'type': 3, 'params': {}, 'element_type': 0}, {'field_id': 105, 'name': 'modi', 'description': '', 'type': 101, 'params': {'dim': 512}, 'element_type': 0}], 'aliases': [], 'collection_id': 447136566584594360, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1}
{'collection_name': 'hcb_ve_test_ask_3', 'auto_id': False, 'num_shards': 1, 'description': 'HCB TEST VE embeddings', 'fields': [{'field_id': 100, 'name': 'id', 'description': '', 'type': 21, 'params': {'max_length': 1000}, 'element_type': 0, 'is_primary': True}, {'field_id': 101, 'name': 'offset', 'description': '', 'type': 5, 'params': {}, 'element_type': 0}, {'field_id': 102, 'name': 'original_url', 'description': '', 'type': 21, 'params': {'max_length': 65535}, 'element_type': 0}, {'field_id': 103, 'name': 'original_width', 'description': '', 'type': 3, 'params': {}, 'element_type': 0}, {'field_id': 104, 'name': 'original_height', 'description': '', 'type': 3, 'params': {}, 'element_type': 0}, {'field_id': 105, 'name': 'modi', 'description': '', 'type': 101, 'params': {'dim': 512}, 'element_type': 0}], 'aliases': [], 'collection_id': 447136566580387143, 'consistency_level': 0, 'properties': {}, 'num_partitions': 1} `


More logs from querycoor
`
[2024/01/24 18:07:15.413 +00:00] [INFO] [querycoordv2/services.go:57] ["show collections request received"] [traceID=2e12d7937c56f49f31e3bedf46c98ede] [collections="[]"]
[2024/01/24 18:07:15.435 +00:00] [INFO] [querycoordv2/services.go:784] ["get replicas request received"] [traceID=b15ad2fe934afffa77d09d26aa192dfa] [collectionID=446032668450742728] [with-shard-nodes=false]
[2024/01/24 18:07:15.435 +00:00] [WARN] [querycoordv2/handlers.go:319] ["failed to get channels, collection not loaded"]
[2024/01/24 18:07:15.435 +00:00] [WARN] [querycoordv2/services.go:821] ["failed to get replica info"] [traceID=b15ad2fe934afffa77d09d26aa192dfa] [collectionID=446032668450742728] [replica=447208019517505555] [error="failed to get channels, collection not loaded: collection=446032668450742728: collection not found"] [errorVerbose="failed to get channels, collection not loaded: collection=446032668450742728: collection not found\n(1) attached stack trace\n  -- stack trace:\n  | github.com/milvus-io/milvus/pkg/util/merr.WrapErrCollectionNotFound\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/merr/utils.go:374\n  | [...repeated from below...]\nWraps: (2) failed to get channels, collection not loaded\nWraps: (3) attached stack trace\n  -- stack trace:\n  | github.com/milvus-io/milvus/pkg/util/merr.wrapWithField\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/merr/utils.go:760\n  | github.com/milvus-io/milvus/pkg/util/merr.WrapErrCollectionNotFound\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/merr/utils.go:372\n  | github.com/milvus-io/milvus/internal/querycoordv2.(*Server).fillReplicaInfo\n  | \t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/handlers.go:320\n  | github.com/milvus-io/milvus/internal/querycoordv2.(*Server).GetReplicas\n  | \t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/services.go:819\n  | github.com/milvus-io/milvus/internal/distributed/querycoord.(*Server).GetReplicas\n  | \t/go/src/github.com/milvus-io/milvus/internal/distributed/querycoord/service.go:385\n  | github.com/milvus-io/milvus/internal/proto/querypb._QueryCoord_GetReplicas_Handler.func1\n  | \t/go/src/github.com/milvus-io/milvus/internal/proto/querypb/query_coord.pb.go:5560\n  | github.com/milvus-io/milvus/pkg/util/interceptor.ServerIDValidationUnaryServerInterceptor.func1\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/interceptor/server_id_interceptor.go:54\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | github.com/milvus-io/milvus/pkg/util/interceptor.ClusterValidationUnaryServerInterceptor.func1\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/interceptor/cluster_interceptor.go:48\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | github.com/milvus-io/milvus/pkg/util/logutil.UnaryTraceLoggerInterceptor\n  | \t/go/src/github.com/milvus-io/milvus/pkg/util/logutil/grpc_interceptor.go:23\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc.UnaryServerInterceptor.func1\n  | \t/go/pkg/mod/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.38.0/interceptor.go:342\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:25\n  | github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1\n  | \t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.3.0/chain.go:34\n  | github.com/milvus-io/milvus/internal/proto/querypb._QueryCoord_GetReplicas_Handler\n  | \t/go/src/github.com/milvus-io/milvus/internal/proto/querypb/query_coord.pb.go:5562\n  | google.golang.org/grpc.(*Server).processUnaryRPC\n  | \t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:1345\n  | google.golang.org/grpc.(*Server).handleStream\n  | \t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:1722\n  | google.golang.org/grpc.(*Server).serveStreams.func1.2\n  | \t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:966\n  | runtime.goexit\n  | \t/usr/local/go/src/runtime/asm_amd64.s:1598\nWraps: (4) collection=446032668450742728\nWraps: (5) collection not found\nError types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) merr.milvusError"]
[2024/01/24 18:07:15.580 +00:00] [INFO] [querycoordv2/services.go:57] ["show collections request received"] [traceID=834599077f20c8752ecee378145928e4] [collections="[]"]
`

**Comment by yhmo** [comment]
Collection count - 73k

So, there are 73000 collections in the cluster?

**Reply by saratkirana** [reply]
Actually No. of collections - 17 (13 in 1 database and 4 in another one). Biggest collection has 487M entries and smallest is 73k entities.

**Comment by xiaofan-luan** [comment]
We'd recommend to use partitionkey for multi tenant rather than create collections.

From our current test, the max collection number we recommend is 10000.

**Reply by saratkirana** [reply]
No. of collections - 17 (13 in 1 database and 4 in another one). Biggest collection has 487M entries and smallest is 73k entities.

**Comment by yhmo** [comment]
I think a bug is triggered inside the server. Search all the WARN and ERROR logs of the querycoord/querynodes/rootcoord during the time of loading.
